src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,4 +63,21 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.title = title;
 
   }
+
+  @Override
+  public boolean equals(Object toCompare)
```

Comment:
Actually, because of research, I added the hashcode() for everything I overwrote equals() for